packages/expo-notifications/android/src/main/java/expo/modules/notifications/NotificationsPackage.java

```diff
@@ -0,0 +1,25 @@
+package expo.modules.notifications;
+
+import android.content.Context;
+
+import org.unimodules.core.BasePackage;
+import org.unimodules.core.ExportedModule;
+import org.unimodules.core.interfaces.SingletonModule;
+
+import java.util.Collections;
+import java.util.List;
+
+import expo.modules.notifications.tokens.PushTokenManager;
+import expo.modules.notifications.tokens.PushTokenModule;
+
+public class NotificationsPackage extends BasePackage {
+  @Override
+  public List<ExportedModule> createExportedModules(Context context) {
+    return Collections.singletonList((ExportedModule) new PushTokenModule(context));
```

Comment:
Is this level of modularisation we aim at, or just a temporary state?

Reply:
I, personally, find it clearer, but I wouldn't impose this as a standard without more approving voices.